src/main/scala/org/broadinstitute/dsde/workbench/sam/service/ResourceService.scala

```diff
@@ -428,29 +436,34 @@ class ResourceService(
     validationErrors.map(_.flatten)
   }
 
-    private def fireGroupUpdateNotification(groupId: WorkbenchGroupIdentity, samRequestContext: SamRequestContext): Future[Unit] =
-    cloudExtensions.onGroupUpdate(Seq(groupId), samRequestContext) recover {
-      case t: Throwable =>
-        logger.error(s"error calling cloudExtensions.onGroupUpdate for $groupId", t)
-        throw t
-    }
+  private def fireGroupUpdateNotification(groupId: WorkbenchGroupIdentity, samRequestContext: SamRequestContext): IO[Unit] =
+    IO.fromFuture(IO(cloudExtensions.onGroupUpdate(Seq(groupId), samRequestContext))).runAsync {
```

Comment:
Note that this is run async which is in line with how it worked before. This is why the tests have timeouts. I also added some timings (see GoogleExtensions) so we can get a sense of how long this actually takes to see if we can make it synchronous. I view this as an integrity issue. If the instance goes down during this process the change has been committed but messages may be dropped. Perhaps a better solution is to have this be a single message then have the group sync machinery do this.